src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
Thanks for the comments!  I'll keep this as-is unless someone else objects.

a) Yes.  Every test fails with referential integrity exception.

b) Subsequent tests begin in an unknown state, so some will fail, somewhat mysteriously.